02_configure_host.sh

```diff
@@ -169,3 +169,14 @@ if [ "$MANAGE_BR_BRIDGE" == "y" ] ; then
     sudo systemctl restart NetworkManager
   fi
 fi
+
+# Create openstack clouds.yaml
+if [[ ! -f "$OPENSTACK_CONFIG" ]]
+then
+  mkdir -p "$(dirname "$OPENSTACK_CONFIG")"
+  echo "clouds:" > "$OPENSTACK_CONFIG"
+fi
+
+# Merge dev-scripts clouds.yaml with user's existing clouds.yaml
+OPENSTACK_CONFIG_MERGED=$(yq -y -s '.[0] * .[1]' "$OPENSTACK_CONFIG" clouds.yaml)
+echo "$OPENSTACK_CONFIG_MERGED" > "$OPENSTACK_CONFIG"
```

Comment:
I noticed this doesn't actually work as expected if you happen to run openstackclient in the dev-scripts directory, because the clouds.yaml there takes precedence over the ~/.config/openstack/clouds.yaml

So we may want to rename the dev-scripts file or remove it?

Reply:
Oh, good point.  How about `clouds.yaml.template`?

Reply:
Or just not do this at all, if the one if $PWD takes precedence, then just use clouds.yaml in the repo. We could even keep them named `metal3` in both repos.

Reply:
Yeah maybe it's just easier to say run openstackclient from inside the respective repos, then each can have a clouds.yaml with $whatever endpoints?

No strong opinion on which approach really, but seems like the $PWD case probably covers the most common case, e.g debugging right after `make` fails

Reply:
Yea, I think that's fine, there's already info in README.md about using the clouds.yaml, so I'll close this and make metal3-dev-env use the same approach